include/simdjson/generic/ondemand/document-inl.h

```diff
@@ -492,6 +503,7 @@ simdjson_really_inline simdjson_result<value> document_reference::find_field_uno
 simdjson_really_inline simdjson_result<value> document_reference::find_field_unordered(const char *key) & noexcept { return doc->find_field_unordered(key); }
 simdjson_really_inline simdjson_result<json_type> document_reference::type() noexcept { return doc->type(); }
 simdjson_really_inline simdjson_result<bool> document_reference::is_scalar() noexcept { return doc->is_scalar(); }
+simdjson_really_inline simdjson_result<const char *> document_reference::current_location() noexcept { return doc->current_location(); };
```

Comment:
I believe ; is redundant and GCC complains it:
```
include/simdjson/generic/ondemand/document-inl.h:530:137: error: extra ';' [-Werror=pedantic]
 simdjson_really_inline simdjson_result<const char *> document_reference::current_location() noexcept { return doc->current_location(); };
```

Reply:
If you have concerns, please issue a PR that we can review.

Reply:
@lemire it would be great if you can review this: https://github.com/simdjson/simdjson/pull/1715